README.md

```diff
@@ -23,11 +23,11 @@
 
 Diffing notebooks in the terminal:
 
-![terminal-diff](docs/source/images/nbdiff-terminal.png)
+![terminal-diff](https://github.com/jupyter/nbdime/raw/master/docs/source/images/nbdiff-terminal.png)
```

Comment:
Won't this break the images in older releases on PyPI if these are moved in master? Ideally the code that puts this inti `setup.py` should get the current git hash and use that to create a permalink?

Reply:
Good point, changed this to current Git hash.